middleware.ts

```diff
@@ -1,9 +1,51 @@
 import { NextResponse } from 'next/server';
-import createMiddleware from './next.middleware';
-import detectLanguage from './middlewares/detectLanguage';
+import { availableLocales } from './next.locales.mjs';
+import type { NextRequest } from 'next/server';
 
-const nextMiddleware = createMiddleware(NextResponse);
+// This Middleware is responsible for handling automatic language detection from an user's Browser
```

Comment:
```suggestion
// This Middleware is responsible for handling automatic language detection from a user's Browser
```

https://english.stackexchange.com/a/105117/9506

Reply:
Ty!